lib/skins.js

```diff
@@ -69,6 +74,33 @@ module.exports = function (gulp, gutil) {
         }
       });
     },
+    vars: function () {
+      var baseFeaturePath = 'res/feature/responsivebase';
+      var peakFeaturePath = 'res/feature/responsivepeak';
+      var baseSassPath = 'res/skins/bootstrap_base/sass';
+      var peakSassPath = 'res/skins/responsive_peak/sass';
+      var varsPath = 'codebook/_variables-*.scss';
+
+      function getSkinVersions(dir) {
+        return fs().readdirSync(dir)
+          .filter(function(file) {
+            return fs().statSync(path().join(dir, file)).isDirectory();
+          });
+      }
+
+      var peakVars = getSkinVersions(peakFeaturePath).map(function(version) {
+        return gulp.src([path().join(baseFeaturePath, version, '**', varsPath),
+            path().join(peakFeaturePath, version, '**', varsPath)])
+          .pipe(replace()('!default', ''))
+          .pipe(replace()(/^\s*(\$.*)/gm, '// $1'))
```

Comment:
You might comment what the before/after of the contents of the file that is changed by these replaces.